cli/consts.py

```diff
@@ -97,3 +99,4 @@
 # git consts
 COMMIT_DIFF_DELETED_FILE_CHANGE_TYPE = 'D'
 GIT_HEAD_COMMIT_REV = 'HEAD'
+EMPTY_COMMIT_SHA = '0000000000000000000000000000000000000000'
```

Comment:
Nothing like that built-in in the git repository that we are using?

Reply:
no